.gitignore

```
@@ -1,3 +1,14 @@
 *tmp/cache*
```

probably better to disallow everything under tmp folder except for `.keep` file
```suggestion
/tmp/*
!/tmp/.keep
```